ci/pipeline.yml

```diff
@@ -0,0 +1,119 @@
+---
+resource_types:
+  - name: gcs-resource
+    type: docker-image
+    source:
+      repository: frodenas/gcs-resource
+
+resources:
+  - name: gcp-service-broker
+    type: git
+    source:
+      uri: https://github.com/GoogleCloudPlatform/gcp-service-broker.git
+      branch: concourse
+      ignore_paths:
+        - ci/*
+
+  - name: gcp-service-broker-ci
```

Comment:
Why 2 resources for the same git repo/branch? I see one ignores the `ci/` dir and the other doesn't. Possible to use just 1 and avoid cloning twice?